config.h

```diff
@@ -1,22 +1,21 @@
-// Release version 1.1
-
-///////////////////////////////////////////
-//////////// PSI CONFIGURATION //////////// 
-///////////////////////////////////////////
+// Release version 1.1.01
 
+///////////////////////////////////////////////////
+//////////// PSI CONFIGURATION ////////////////////
+///////////////////////////////////////////////////
 
 
-////////////////////////////////////////
-//////////// Timer Settings ///////////
-//////////////////////////////////////
+///////////////////////////////////////////////////
```

Comment:
Nit. The // on left should match those on right.  This addition makes it uneven.  Not a fan.